nixos/modules/system/boot/loader/grub/grub.nix

```diff
@@ -58,7 +58,7 @@ let
       path = (makeBinPath ([
         pkgs.coreutils pkgs.gnused pkgs.gnugrep pkgs.findutils pkgs.diffutils pkgs.btrfs-progs
         pkgs.utillinux ] ++ (if cfg.efiSupport && (cfg.version == 2) then [pkgs.efibootmgr ] else [])
-      )) + ":" + (makeSearchPathOutputs "sbin" ["bin"] [
+      )) + ":" + (makeSearchPathOutput "bin" "sbin" [
```

Comment:
I thought of looking through all our `sbin` uses and replace them, but we have too much of them: `grep -r 'sbin' | wc -l` shows 835 occurrences. I think removing them is out of scope of this PR, although I can remove those that I touched if desired.